osm2gtfs/osm2gtfs.py

```diff
@@ -44,58 +46,62 @@ def main():
     elif args.refresh_stops:
         data.get_stops(refresh=True)
         sys.exit(0)
+    elif args.refresh_schedule_source:
+        config.get_schedule_source(refresh=True)
+        sys.exit(0)
```

Comment:
Why do you `sys.exit(0)` here? Couldn't the script continue to run as in the case of `args.refresh_all`?

Reply:
Hmm, this is a good question. I follow your point, removed all `sys.exit(0)` from the other calls also.